lib/apiservers/engine/backends/backends.go

```diff
@@ -212,8 +196,18 @@ func ProductVersion() string {
 	return productVersion
 }
 
+// VchConfig gets the VIC appliance VM's configspec
 func VchConfig() *config.VirtualContainerHostConfigSpec {
-	return vchConfig
+	var vchConfig config.VirtualContainerHostConfigSpec
```

Comment:
We don't know yet.  It was being hashed out the last time I heard.  It maybe Admiral.